lms/views/api/blackboard/files.py

```diff
@@ -23,10 +23,26 @@ def __init__(self, request):
     def list_files(self):
         """Return the list of files in the given course."""
 
-        return self.blackboard_api_client.list_files(
+        files = self.blackboard_api_client.list_files(
             self.request.matchdict["course_id"]
         )
 
+        pdf_files = []
+
+        for file in files:
+            if file.get("mimeType") != "application/pdf":
+                continue
+
+            pdf_files.append(
+                {
+                    "id": f"blackboard://content-resource/{file['id']}/",
+                    "display_name": file["name"],
+                    "updated_at": file["modified"],
+                }
+            )
```

Comment:
This loop the view doing what the schema uses to

Reply:
:+1:  I think I like this approach better